zipkin-ui/js/component_ui/traceToMustache.js

```diff
@@ -181,8 +191,6 @@ export default function traceToMustache(v2Trace, logsUrl = undefined) {
   const traceTimestamp = trace[0].timestamp || 0;
   const spanDepths = toSpanDepths(trace);
```

Comment:
fyi the logic below I will rewrite in a different PR after I figure out what it is doing.